### DISCUSSION TITLE
How to enable NSFW detection

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [photoprism/photoprism#3249] [Q&A]
I don't have a defaults.yml but an options.yaml
looking at the instructions it's 
DetectNSFW | bool | --detect-nsfw

stupid question but how do you format that in that file? like the below???

Experimental: false
JpegQuality: 85
ReadOnly: false
DetectNSFW: --detect-nsfw

**Comment by lastzero** [comment]
true

**Reply by NorthernMunkey** [reply]
ok thank you, are you able to kick off a nsfw scan on an already imported photo database? and slightly differently, a recheck of face recognition?

**Reply by lastzero** [reply]
The private flag is only set once for new pictures. Face recognition runs continuously.

**Reply by lastzero** [reply]
Should be documented. If not, feel free to help improve our documentation.

**Reply by NorthernMunkey** [reply]
thank you i'll have a look, i'll rescan everything back in then, so it can categorise them
